doc/changes/changes_3.0.4.md

```diff
@@ -4,16 +4,25 @@ Code name:
 
 ## Summary
 
+## Bugfixes
```

Comment:
Do you plan to release it?
Since it's a security fix we probably should do a release.
If so add a code name, summary and release date

Reply:
thanks, yes, sure, I planned this as my next action anyway.
Let me prepare I will come back for a review before triggering the release.